### PR TITLE
Update API docs to add relationship of swarm with docker api and clarify from missing endpoint

### PR DESCRIPTION
**Pull request by allencloud**
This pr did 3 things:
1. add relationship of Swarm version and Docker API version supported;
2. clarify missing endpoint in swarm corresponding API version
3. add endpoints behaved differently.

Signed-off-by: Sun Hongliang allen.sun@daocloud.io

### CODE REVIEW

**Comment by nishanttotla** [comment]
I'd rather not delete the whole section. Instead replace with some other endpoints that are missing. In general, we've not updated that part very well in the past.

**Comment by allencloud** [comment]
@nishanttotla 
Great, Reasonable.
Maybe I can help to do the comparison soon.

**Comment by SvenDowideit** [comment]
docs LGTM

**Comment by allencloud** [comment]
ping @sfsmithcha, any update?

**Comment by jimmyxian** [comment]
Remove same as https://github.com/docker/swarm/pull/2040

File: docs/swarm-api.md [hunk 1]
```diff
@@ -19,11 +19,8 @@ Remote API.
 
 ## Missing endpoints
 
-Some endpoints have not yet been implemented and will return a 404 error.
-
-```
-POST "/images/create" : "docker import" flow not implement
-```
+Now Docker API that Swarm supports is version 1.22 or lower. In API version 1.22,
```
Comment (sfsmithcha) [inline docs/swarm-api.md]: Please edit for clarity:

As of Swarm version 1.2.1, Swarm supports the API version 1.22 or lower. There are no missing endpoints for Swarm in API version 1.22.

**Comment by sfsmithcha** [comment]
@allencloud , need to clarify from which point/version Swarm supports the API. Please see my comment.

File: docs/swarm-api.md (hunk 1 above)
Comment (allencloud) [inline docs/swarm-api.md]: Hi，@sfsmithcha, totally reasonable, I will check and update that soon.

**Comment by allencloud** [comment]
@sfsmithcha @SvenDowideit @jimmyxian 
I did more change. Updated the PR topic name name and PR description.

**Comment by sfsmithcha** [comment]
@allencloud , thanks for all your work on this. however, these changes introduce some editorial issues for the doc set. i think rather than asking you to make the changes, I probably need to take an editorial pass.

For usability purposes, I think I will want to keep only information for the latest version(s) in this doc and add a link to a separate doc that details the versioning issues.

I'm fairly swamped with another project right now. Feel free to ping me in a week or so.

**Comment by allencloud** [comment]
ping @sfsmithcha updated the PR to update docs about API. PTAL

ping @nishanttotla @vieux @abronan @dongluochen 
In the part of **Endpoints which behave differently**, may be we should check PR #2299 #2297 #2295 #2275 #2261

**Comment by allencloud** [comment]
PTAL @SvenDowideit @sfsmithcha

**Comment by nishanttotla** [comment]
Hey @allencloud, do you want to still get this PR in? If so, could you make sure if it needs updating?

**Comment by mstanleyjones** [comment]
Is this PR worth keeping open? If so, I can review it and try to help.

**Comment by nishanttotla** [comment]
Ping @allencloud

**Comment by allencloud** [comment]
Thanks a lot @mstanleyjones welcome your review.
And PR updated @nishanttotla

**Comment by thaJeztah** [comment]
Oh boy, looks like this wasn't merged before the docs were moved; can you reopen this against the central docs repository https://github.com/docker/docker.github.io/ if it's still something that has to be addressed?

**Pull request closed without merging**